pgrx-examples/decoder/README.md

```diff
@@ -0,0 +1,119 @@
+
+# A simple Change Data Capture ([CDC]) extension.
+
+This extension will extract the [DML] changes from Postgres WAL
+using [Logical Decoding] and export them in JSON format using [serde].
+
+## Principles
+
+* Postgres triggers various callbacks at the different stages of a transaction
+* The decoder defines some of these callbacks : begin, change, commit, etc.
+* The callbacks extract the changes made during the transaction
+* They build Rust structs (Action, Tuple) to represent those changes
+* The structs are then serialized into JSON
+* The JSON output is sent into a replication slot
+* The ouput be can consumed in various ways by a remote client.
+
+
+## Requirements
+
+In order to use this extension with a cargo-pgrx managed database, you'll
+need to add the configuration below in "$PGRX_HOME/data-$PGVER/postgresql.conf".
```

Comment:
isn't this "any" database, to the appropriate postgresql.conf?

Reply:
I think I meant "cargo-pgrx managed instance"